lib/helpers.js

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+
+// FIXME: is this right? Why is the host coming from the request instead
+// of the bedrock config?
```

Comment:
Seems like it should come from the bedrock config. We'd only need to take it from the request if we were going to support multiple hosts on a server -- and if we could be guaranteed to trust that host. I don't think we have that sort of use case anymore?